vector/src/main/java/im/vector/app/features/settings/devices/v2/DeviceUserAgent.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.settings.devices.v2
+
+import im.vector.app.features.settings.devices.v2.list.DeviceType
+
+data class DeviceUserAgent(
```

Comment:
Maybe we should name it `DeviceExtendedInfo` to avoid referencing the source of the data? We could decide in the future to use something different than the user agent but the info will still be the same.

Reply:
Make sense, done.